.github/workflows/deploy-api-oc4.yml

```diff
@@ -0,0 +1,46 @@
+name: Build Efiling Reviewer API Image and Push to Openshift Registry
+
+on:
+  push:
+    branches: [main]
+    paths:
+      - "src/backend/**"
+      - ".github/workflows/deploy-api-oc4.yml"
+
+  # for testing ...
+  pull_request:
+    paths:
```

Comment:
missing branches here

Reply:
removed pull_request ... was for testing